biolink-model.yaml

```diff
@@ -8014,6 +8153,8 @@ classes:
         range: genomic entity  ## todo protein
       object:
         range: genomic entity ## typically a chromosome
+    exact_mappings:
```

Comment:
Better to call the prefix `CHADO`. But I'll defer to @cmungall to chime in on the ideal choice.

Also, the prefix needs to be defined in the YAML.

Reply:
OK. I just used the prefix from the original PR. Doesn't seem to have an associated URI (I'll dig for this)